### ISSUE TITLE
docs: Update general docs README

### ISSUE THREAD

**Issue by shmorri** [JanssenProject/jans#1950]
https://github.com/JanssenProject/jans/blob/main/docs/README.md

This file is already out of date and needs to be updated with final content.

**Comment by ossdhaval** [comment]
@shmorri Right. And I am not sure if this is needed now or not in presence of [jans.io/docs](jans.io/docs). 
Intention behind this page was that it provides a TOC for people using GH when we didn't had static site. 
_Does maintaining this make sense anymore?_
